src/openforms/submissions/form_logic.py

```diff
@@ -7,7 +7,6 @@
 
 from ..forms.models.form import FormLogic
 from ..prefill import JSONObject
-from .models import Submission, SubmissionStep
```

Comment:
please replace this with a conditional import, otherwise linters will complain:

```py
from typing import TYPE_CHECKING

...

if TYPE_CHECKING:
    from .models import Submission, SubmissionStep
```